test/data/records.json

```diff
@@ -0,0 +1,6 @@
+[
+    {"lfn":"/store/file.root",
+     "pfn":"test/data/testdata.txt",
+     "block":"abc-123",
```

Comment:
I did same as mentioned in wiki pages. We need to change the block convention over there.